f/frizzle/tests/conftest.py

```diff
@@ -0,0 +1,20 @@
+import pytest
+import responses
```

Comment:
From the PR description:

> It introduces some indirection that I don't love. gc-scripts-hub is already unconventional in that tox.ini builds separate environments different test suites, since their dependencies can diverge so much. Under this PR, one test suite may use the plugin that introduces its own package dependencies but those dependencies are not declared at the shared library (plugin) scope. Rather the test suite needs to be aware of which shared fixture(s) it uses -- and that fixture's package dependencies -- and declare them in its own `requirements-test.txt`.

This line is an example.  Merely pulling in this file will give an `ImportError` unless the test suite itself has opted to install `responses` and `testing.postgresl`.


This could be ameliorated by pulling the `import` into the fixture itself, so that a test only needs to install the fixtures it opts to use. Dunno if that solution is good enough (I'm ok with it for now..) or is just  a warning sign of things to come.